code/datums/setup_option/core_implants.dm

```diff
@@ -91,7 +93,6 @@
 	particular design is an opifex original and one of the best that can be found in the galaxy."
 	implant_organ_type = "opifex nanogate"
 	restricted_jobs = list(
-		/datum/job/outsider, // Nanogates are only available to colonist or allies.
```

Comment:
I accually agree with this one

Reply:
Aren't opifex the ones who produce nanogates natively and thus should have access to them, in theory

Reply:
Ye thats why i accually agree on this one

Reply:
they are disagreeing on the rest. Bird gate = good 
rest = bad
 is the word of the cat.
 
 Personally I don't think its to big a deal. The biggest buff is access to healing something as is outsiders already have with combat medkits and such. Personally I'd likely even take other buffs over the healing with a standard gate given those combat medkits are handy.